src/server/response.cpp

```diff
@@ -133,14 +132,14 @@ void Response::set_kind(Kind k)
     m_etag.set_option(ETag::ZIM_CONTENT);
 }
 
-std::unique_ptr<Response> Response::build(const InternalServer& server)
+std::unique_ptr<Response> Response::build()
 {
-  return std::unique_ptr<Response>(new Response(server.m_verbose.load()));
+  return std::unique_ptr<Response>(new Response());
```

Comment:
`std::make_unique` ? (Now we have moved to C++14 (and more))

Reply:
Done in a separate commit